src/portable/portable.h

```diff
@@ -21,13 +21,14 @@ typedef struct PortableMetadata {
 #define PORTABLE_METADATA_IS_UNIT(m) (!IN_SET((m)->name[0], 0, '/'))
 
 typedef enum PortableFlags {
-        PORTABLE_RUNTIME        = 1 << 0, /* Public API via DBUS, do not change */
-        PORTABLE_PREFER_COPY    = 1 << 1,
-        PORTABLE_PREFER_SYMLINK = 1 << 2,
-        PORTABLE_REATTACH       = 1 << 3,
-        _PORTABLE_MASK_PUBLIC   = PORTABLE_RUNTIME,
+        PORTABLE_RUNTIME                    = 1 << 0,
```

Comment:
the ` /* Public API via DBUS, do not change */` comment was lost here?

Reply:
Moved down - it's an enum, I don't think it's useful to duplicate it, anything up to the marker is fixed, anything after the marker is free game